libraries/DataFlash/LogStructure.h

```diff
@@ -873,6 +873,18 @@ struct PACKED log_Proximity {
     float closest_dist;
 };
 
+struct PACKED log_SRTL {
+    LOG_PACKET_HEADER;
+    uint64_t time_us;
+    uint8_t active;
+    uint16_t num_points;
```

Comment:
These aren't uint16_t's elsewhere they are int16_t and are being type converted for logging only. They should be consistent (and given that it's used for a counter it's probably better to change the definition in SafeRTL to be uint16_t).